python-sdk/src/astro/files/locations/google/gcs.py

```diff
@@ -43,6 +63,23 @@ def size(self) -> int:
             object_name = object_name[1:]
         return int(self.hook.get_size(bucket_name=bucket_name, object_name=object_name))
 
+    def databricks_auth_settings(self) -> dict:
+        """
+        Required settings to upload this file into databricks. Only needed for cloud storage systems
+        like S3
+        :return: A dictionary of settings keys to settings values
+        """
+        creds_dict = self.hook.get_connection(self.conn_id).extra_dejson
```

Comment:
@dimberman, out of curiosity, did you try this with the google default connection when the user is not using a service account JSON?

Reply:
Yes @kaxil and I looked into it. I have added more detailed documentation of what requirements people will need to use this feature.